src/main/java/dev/j3fftw/headlimiter/Utils.java

```diff
@@ -100,7 +102,24 @@ public static void onCheck(Player player, Block block, int maxAmount, int count,
             });
 
             BlockStorage.clearBlockInfo(block.getLocation());
-            player.sendMessage(ChatColor.RED + "You hit the limit of Cargo nodes in this chunk");
+            if (isPlacingRestricted) {
+                player.sendMessage(ChatColor.RED + "You can't place Cargo nodes in unprotected areas!");
```

Comment:
```suggestion
                player.sendMessage(ChatColor.RED + "You can't place Cargo nodes in unclaimed areas!");
```
This might be a better name but thats up to you @Sfiguz7